go-runtime/ftl/ftltest/mock.go

```diff
@@ -0,0 +1,42 @@
+package ftltest
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/TBD54566975/ftl/go-runtime/ftl"
+	"github.com/TBD54566975/ftl/internal/rpc"
+
+	"github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1/ftlv1connect"
+)
+
+type mockFunc func(ctx context.Context, req any) (resp any, err error)
+
+// mockProvider keeps a mapping of verb references to mock functions.
+//
+// It implements the CallOverrider interface to intercept calls with the mock functions.
+type mockProvider struct {
```

Comment:
Would it be worth renaming this to mockVerbProvider in case we want to support mocking of entities other than verbs down the road?